common/config/rush/experiments.json

```diff
@@ -9,7 +9,7 @@
    * By default, 'rush install' passes --no-prefer-frozen-lockfile to 'pnpm install'.
    * Set this option to true to pass '--frozen-lockfile' instead for faster installs.
    */
-  "usePnpmFrozenLockfileForRushInstall": true,
+  // "usePnpmFrozenLockfileForRushInstall": true,
```

Comment:
Intentional change?

Reply:
Nope, edited this for testing. Will revert back.